pkg/services/serviceaccounts/models.go

```diff
@@ -60,3 +60,27 @@ type ServiceAccountProfileDTO struct {
 	Teams         []string        `json:"teams" xorm:"-"`
 	AccessControl map[string]bool `json:"accessControl,omitempty" xorm:"-"`
 }
+
+type ServiceAccountFilter string // used for filtering
+
+const (
+	FilterOnlyExpiredTokens ServiceAccountFilter = "expiredTokens"
+	FilterIncludeAll        ServiceAccountFilter = "all"
+)
+
+type SearchOrgServiceAccountsQuery struct {
+	OrgID  int64
+	Query  string
+	Page   int
+	Limit  int
+	Filter ServiceAccountFilter
+
+	User *models.SignedInUser
+}
+
+type SearchOrgServiceAccountsQueryResult struct {
+	TotalCount      int64                `json:"totalCount"`
+	ServiceAccounts []*ServiceAccountDTO `json:"orgServiceAccounts"`
+	Page            int                  `json:"page"`
+	PerPage         int                  `json:"perPage"`
+}
```

Comment:
It doesn't seem used, is it?
```suggestion
```

Reply:
thissss is true! well done to find this. This was part of a bigger refactoring that you too did and remove the query.
I missed to remove them! nice work.

Curious did your ide or just your big brain see this ? 🧠